web/src/features/charts/bar-breakdown/BarBreakdownEmissionsChart.tsx

```diff
@@ -77,57 +83,44 @@ function BarBreakdownEmissionsChart({
   };
 
   return (
-    <svg className="w-full overflow-visible" height={height}>
-      <Axis formatTick={formatTick} height={height} scale={co2Scale} />
-      <g transform={`translate(0, ${productionY})`}>
-        {productionData.map((d, index) => (
-          <Row
-            key={d.mode}
-            index={index}
-            label={t(d.mode)}
-            width={width}
-            scale={co2Scale}
-            value={Math.abs(d.gCo2eq)}
-            onMouseOver={(event) => onProductionRowMouseOver(d.mode, data, event)}
-            onMouseOut={onProductionRowMouseOut}
-            isMobile={isMobile}
-          >
-            <ProductionSourceLegend electricityType={d.mode} />
-            <HorizontalBar
-              className="production"
-              fill={modeColor[d.mode]}
-              range={[0, Math.abs(d.gCo2eq)]}
-              scale={co2Scale}
-            />
-          </Row>
-        ))}
-      </g>
-      <g transform={`translate(0, ${exchangeY})`}>
-        {exchangeData.map((d, index) => (
-          <Row
-            key={d.zoneKey}
-            index={index}
-            label={d.zoneKey}
-            width={width}
-            scale={co2Scale}
-            value={d.exchange}
-            onMouseOver={(event) => onExchangeRowMouseOver(d.zoneKey, data, event)}
-            onMouseOut={onExchangeRowMouseOut}
-            isMobile={isMobile}
-          >
-            <g transform={`translate(-2, 0)`}>
-              <CountryFlag zoneId={d.zoneKey} className="pointer-events-none" />
-            </g>
-            <HorizontalBar
-              className="exchange"
-              fill={'gray'}
-              range={[0, d.gCo2eq]}
+    <>
+      <svg className="w-full overflow-visible" height={height}>
+        <Axis formatTick={formatTick} height={height} scale={co2Scale} />
+        <g transform={`translate(0, ${productionY})`}>
+          {productionData.map((d, index) => (
+            <Row
+              key={d.mode}
+              index={index}
+              label={t(d.mode)}
+              width={width}
               scale={co2Scale}
-            />
-          </Row>
-        ))}
-      </g>
-    </svg>
+              value={Math.abs(d.gCo2eq)}
+              onMouseOver={(event) => onProductionRowMouseOver(d.mode, data, event)}
+              onMouseOut={onProductionRowMouseOut}
+              isMobile={isMobile}
+            >
+              <ProductionSourceLegend electricityType={d.mode} />
+              <HorizontalBar
+                className="production"
+                fill={modeColor[d.mode]}
+                range={[0, Math.abs(d.gCo2eq)]}
+                scale={co2Scale}
+              />
+            </Row>
+          ))}
+        </g>
+      </svg>
```

Comment:
Maybe we should break out this into it's own component as well just as the BarEmissionExchangeChart?

Reply:
I think we should spend some time renaming the charts soon, but for now I have tried to follow the naming convention that we have